docs/core/run-time-config/globalization.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Globalization config settings
+description: Learn about run-time settings that configure globalization aspects of a .NET Core app, for example, how it parses Japanese dates.
+ms.date: 11/27/2019
+ms.topic: reference
+---
+# Run-time configuration options for globalization
+
+## Invariant mode
+
+- Determines whether a .NET Core app runs in globalization invariant mode without access to culture-specific data and behavior or whether it has access to cultural data.
+- Default: Run the app with access to cultural data (`false`).
+- For more information, see [.NET Core globalization invariant mode](https://github.com/dotnet/corefx/blob/master/Documentation/architecture/globalization-invariant-mode.md).
+
+| | Setting name | Values |
+| - | - | - |
+| **runtimeconfig.json** | `System.Globalization.Invariant` | `false` - access to cultural data<br/>`true` - run in invariant mode |
+| **Environment variable** | `DOTNET_SYSTEM_GLOBALIZATION_INVARIANT` | 0 - access to cultural data<br/>1 - run in invariant mode |
```

Comment:
Should the 0 and 1 in all of these (even where it lists the value: Default (0) ) be in enclosed in code elements? Since it's the value to enter, I think it should be on par with false or true.